src/client/auth/auth.proto

```diff
@@ -157,17 +157,30 @@ message SetConfigurationRequest {
 }
 message SetConfigurationResponse {}
 
+// AdminGrant reflects any cluster-wide (admin) permissions a principal has.
+// Principals can have multiple admin scopes.
+message AdminGrant {
+  enum Scope {
+    // SUPER gives access to all APIs and owner on everything in PFS (previous just called admin)
+    SUPER = 0;
+
+    // FS gives Owner on all repos in PFS but not access to manage other aspects of the cluster
+    FS = 1;
+  }
+  repeated Scope scopes = 1;
+}
+
 // Get the current list of cluster admins
 message GetAdminsRequest{}
 message GetAdminsResponse{
-  // admins contains the list of cluster admins
-  repeated string admins = 1;
+  // admins contains a mapping of principals to admin grants 
+  map<string, AdminGrant> admins = 1;
```

Comment:
I think if we want to change the field, we should mark field 1 reserved and add a new field (it's a protobuf convention to avoid serialization errors across versions—https://developers.google.com/protocol-buffers/docs/proto3#reserved explains the rationale).

Also, I think we might need to update our other language clients (JS, Python, Ruby, etc). We maybe don't need to do it in this PR, but I think we'll need to do it before 1.11, so we should make sure to track it. Also, I think the only thing to do on our end is tell Customer Team, but users using those libraries will have to update their clients along with the update to 1.11.